hasjob/templates/confirm_email.html

```diff
@@ -0,0 +1,28 @@
+<div itemscope itemtype="http://schema.org/EmailMessage">
```

Comment:
In Hasjob, we're using an email template wrapper so we can add headers and footers across all emails. The `confirm_email.md` template is an older template that got left out. Since you've converted it to HTML, can you use the wrapper now?

```jinja2
{% extends "inc/email_layout_lite.html" %}
{% block content %}
  <!-- Content comes here -->
{% endblock %}
```